x-pack/legacy/plugins/security/public/hacks/legacy.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { uiModules } from 'ui/modules';
+import { npSetup, npStart } from 'ui/new_platform';
+import routes from 'ui/routes';
+import { isSystemApiRequest } from '../../../../../../src/plugins/kibana_legacy/public';
+import { SecurityPluginSetup } from '../../../../../plugins/security/public';
+
+const securityPluginSetup = (npSetup.plugins as any).security as SecurityPluginSetup;
+if (securityPluginSetup) {
+  routes.when('/account', {
```

Comment:
__note:__ for BWC reasons, in a new plugin `Account Management` view uses `/security/account` URL instead of old `/kibana#account`. Not sure what would be the best URL honestly. Technically it's recommended to host our views under `/security/....`, but many apps/views we have are kind of special and don't respect that. If `/security/account` sounds good to you I can move under `/security` few more views namely `/logged_out` and `/overwritten_session` keeping only `/login` and `/logout` under the root path.

Reply:
Good, will move `/logged_out` and `/overwritten_session` under `/security` as well.